libeos-payg/manager.c

```diff
@@ -335,6 +336,8 @@ epg_manager_constructed (GObject *object)
     self->clock = EPG_CLOCK (epg_real_clock_new ());
 
   self->account_id = g_strdup ("");
+  if (self->account_id_file == NULL)
+    self->account_id_file = g_file_new_for_path (PREFIX "/local/share/eos-payg/device-id");
```

Comment:
Since the account ID is derived from the machine ID – why not read the first N characters of `/etc/machine-id`?

Reply:
Because the machine-id could have changed since (although we would hope it did not). Also, it seems better to keep it together with other eos-paygd's data.